repos/system_upgrade/el8toel9/actors/networkmanagerconnectionscanner/libraries/networkmanagerconnectionscanner.py

```diff
@@ -0,0 +1,65 @@
+import errno
+import os
+
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common import utils
+from leapp.libraries.stdlib import api
+from leapp.models import NetworkManagerConnection, NetworkManagerConnectionProperty, NetworkManagerConnectionSetting
+
+libnm_available = False
+err_details = None
+try:
+    import gi
+    try:
+        gi.require_version("NM", "1.0")
+        from gi.repository import GLib, NM
+        libnm_available = True
+    except ValueError:
+        err_details = 'NetworkManager-libnm package is not available'
+except ImportError:
+    err_details = 'python3-gobject-base package is not available'
```

Comment:
@lkundrak  I need to sync  with you about this as probably the package should have updated requirement. Let's sync on monday.

Reply:
@lkundrak unfortunately I do not have permissions to update your branch. So I pushed the changes to my fork. Please update your branch when you are online, so we can merge it: https://github.com/pirat89/leapp-repository/tree/lr/ifcfg-nm-rework